### DISCUSSION TITLE
Unable to delete or add rules

### DISCUSSION THREAD

**Discussion by RickNas** [firefly-iii/firefly-iii#8093] [Firefly III]
Hi all,

I have Firefly III 5.4.6 installed on my Synology NAS using Docker.
For some reason I am unable to create new or deleting the default rule. I have tried different ways, but it looks like as it is "read only".

I can import CSVs and the transactions will appear in the in the app. I can also manually convert the transaction type.

I am new to Firefly, is this something some people have seen before?

Thanks

**Comment by JC5** [comment]
Is there a way for you to upgrade to the latest version?

**Reply by RickNas** [reply]
Thanks JC5. I will give it a try later today.

**Reply by RickNas** [reply]
I am having problems making the latest version initiate, but I will open a new thread for that.
Once again, thanks for the support.
